### PR TITLE
Location header in proxy

### LINKED ISSUE #82: Section 2.5 - more precision around the Location header contents
Per "The "Location" header must be rewritten", it would be useful to describe the new target.

### PR DESCRIPTION
**Pull request by yaronf**
Fixes #82.

### CODE REVIEW

File: STAR-Delegation/draft-ietf-acme-star-delegation.md [hunk 1]
```diff
@@ -528,11 +528,11 @@ ACME Delegation process:
   * Similarly, the `star-certificate` URL MUST be copied as-is.
   * The `finalize` URL is rewritten, so that the `finalize` request will be
     made to the proxy.
-  * The `Location` header must be rewritten.
+  * The `Location` header MUST be rewritten to point to an `Order` object on the proxy.
```
Comment (thomas-fossati) [inline STAR-Delegation/draft-ietf-acme-star-delegation.md:531]: there is another instance of this in the new-order response section above

Comment (thomas-fossati) [inline STAR-Delegation/draft-ietf-acme-star-delegation.md:531]:
```suggestion
  * The `Location` header MUST be rewritten to point to an Order object on the proxy.
```

(We don't seem to quote Object in other parts of the document.)

File: STAR-Delegation/draft-ietf-acme-star-delegation.md [hunk 2]
```diff
@@ -528,11 +528,11 @@ ACME Delegation process:
...
 * Finalize request:
   * The CSR MUST be copied as-is.
 * Finalize response:
-  * Both the `Location` header and the `finalize` URLs are rewritten.
+  * Both the `Location` header and the `finalize` URLs are rewritten as for Get Order.
```
Comment (thomas-fossati) [inline STAR-Delegation/draft-ietf-acme-star-delegation.md:535]: Just a thought: should "Get Order" be "POST-as-GET order" instead (as per 8555)?

Comment (yaronf) [reply STAR-Delegation/draft-ietf-acme-star-delegation.md:535]: No, IMO this is a logical name of the request, not the actual HTTP method that's being used.

**Review by thomas-fossati — APPROVED** [review]
This is good.  However, another HTTP thing that might need masking is the Link header.

Now tracked by #116

**Pull request merged**